packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -12,7 +12,7 @@ export const tour_step_config: Step[] = [
                 <div className='toggle-account-type__divider' />
             </Text>
         ),
-        content: <Text as='p'>{localize('Switch between your demo and real accounts.')}</Text>,
+        content: <Text as='p'>{localize('You can switch between real and demo accounts.')}</Text>,
```

Comment:
please apply these changes in the `https://github.com/matin-binary/deriv-app/pull/73/files` pr and close this one